### PR TITLE
Add hubspot_form_ids to JS config

### LINKED ISSUE #12932: Add hubspot form newsletter id to /me endpoint request
In public maps, we have an input to subscribe to the newsletter. This input is generated by hubspot, but it needs a token to render it.

captura de pantalla 2017-10-10 a las 12 21 02

This is what we currently have at `/api/v3/me`:

captura de pantalla 2017-10-10 a las 12 23 37

* Form id in `app_config.yml`: https://github.com/CartoDB/cartodb/blob/master/config/app_config.yml.sample#L89
* Public map footer:
https://github.com/CartoDB/cartodb/blob/master/app/views/admin/shared/_public_footer.html.erb#L13
* Hubspot form helper
https://github.com/CartoDB/cartodb/blob/master/app/helpers/application_helper.rb#L113

I was thinking the endpoint should return this value in 'hubspot_ids' *stringified* json, what do you think?

### PR DESCRIPTION
**Pull request by amartinj**
Fixes #12932

### CODE REVIEW

File: app/helpers/frontend_config_helper.rb [hunk 1]
```diff
@@ -42,6 +42,7 @@ def frontend_config_hash(user = current_user)
     if CartoDB::Hubspot::instance.enabled? && !CartoDB::Hubspot::instance.token.blank?
       config[:hubspot_token] = CartoDB::Hubspot::instance.token
       config[:hubspot_ids] = CartoDB::Hubspot::instance.event_ids.to_json.html_safe
+      config[:hubspot_form_ids] = Cartodb.get_config(:metrics, 'hubspot', 'form_ids')
```
Comment (juanignaciosl) [inline app/helpers/frontend_config_helper.rb]: Please hide the configuration like the other two values do, for consistency.

**Comment by amartinj** [comment]
retest this please

**Comment by elenatorro** [comment]
retest this please

**Comment by javitonino** [comment]
Works in staging, with Hubspot enabled and disabled.

**Pull request merged**